storybookjs/storybook
# use @storybook published deprecated dependencies

Issue: temporarily relieves #2303 

## What I did

forked all the komposer/mantra deprecated packages and published version without peer dependency warning

## How to test

Is this testable with jest or storyshots?
n
Does this need a new example in the kitchen sink apps?
n
Does this need an update to the documentation?
n
If your answer is yes to any of these, please make sure to include it in your PR.
